recipe/meta.yaml

```diff
@@ -1,36 +1,71 @@
 {% set name = "maxminddb" %}
-{% set version = "2.4.0" %}
+{% set version = "2.5.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 81e54e53408bd502650e5969ccba16780af659ec1db1c44b2c997e4330a5ed96
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 4807d374e645bd68334e4f487ba85a27189dbc1267a98e644aa686a7927e0559
+    patches:
+      - patches/0001-Remove-runtime-dependency-on-setuptools.patch
+
+  # Because of https://github.com/maxmind/MaxMind-DB-Reader-python/blob/v2.5.1/.gitmodules#L6
+  # we can't use git_url. It fails on prefect because we don't have a proper ssh key setup.
+  - url: https://github.com/maxmind/MaxMind-DB-Reader-python/archive/refs/tags/v2.5.1.tar.gz
+    sha256: fe6ce9978ade80246f5b43aa70b5f7dd333e4423b1fe04a411169d4e8d236b71
+    folder: git-repo
+
+  - url: https://github.com/maxmind/MaxMind-DB/archive/a75bfb17a0e77f576c9eef0cfbf6220909e959e7.zip
```

Comment:
just wondering why you're using an older commit?

Reply:
Because that's the commit that the submodule is pinned at. See https://github.com/maxmind/MaxMind-DB-Reader-python/tree/v2.5.1/tests.

Reply:
ah. thanks